uuid-kinds/src/lib.rs

```diff
@@ -57,6 +57,7 @@ impl_typed_uuid_kind! {
     LoopbackAddress => "loopback_address",
     OmicronZone => "service",
     PhysicalDisk => "physical_disk",
+    Propolis => "propolis",
```

Comment:
In some places, we refer to the UUID of a Propolis process as a "Propolis ID", elsewhere, we call it a "VMM ID". Perhaps we should commit to being more consistent with naming and always refer to them as one or the other (at least, when it doesn't require a database migration...)

Reply:
The general rule I've tried to apply is to use "Propolis ID" to refer to these UUIDs, except when working directly with DB queries where we're working with the primary key to the VMM table.

(When I created the VMM table in #4194, `propolis_id` was a much more common field in slog messages that contained a Propolis/VMM ID, so when it came time to choose an option it won. IMO the structured logging field name should carry a lot of weight--it should be easy to filter for messages concerning a single VMM without having to write `propolis_id = "foo" || vmm_id = "foo"` to get all the messages.)

Reply:
Yup, that makes sense. I may have to rename some `vmm_id` fields in logs I've added...